.eslintrc.js

```diff
@@ -1,25 +1,20 @@
 module.exports = {
   root: true,
 
-  extends: [
-    '@metamask/eslint-config',
-    '@metamask/eslint-config/config/nodejs',
-  ],
+  extends: ['@metamask/eslint-config'],
 
-  parserOptions: {
-    ecmaVersion: 2017,
-    ecmaFeatures: {
-      arrowFunctions: true,
-      classes: true,
+  overrides: [
+    {
+      files: ['*.js'],
+      extends: ['@metamask/eslint-config-nodejs'],
+      env: {
+        es2020: true,
```

Comment:
I bumped this from ES2017 to ES2020 for object spread support. That said, is there a reason why we've been using ES2017? Node v14 is very, very close to ES2020 in terms of compatibility ([source](https://node.green/#ES2020)) so it feels like a better fit.

Reply:
Alright 😞

Reply:
Once we compile TS to JS, then this is okay to bump, though, right? Because this just affects what ESLint sees, not what the final output is.

Reply:
Yes, in fact that's already set in the TS ESLint config we use

Reply:
Right, gotcha 👍🏻